packages/imagemagick7.rb

```diff
@@ -51,19 +45,20 @@ def self.patch
   end
 
   def self.build
-    system "CFLAGS=' -I#{CREW_PREFIX}/include/gdk-pixbuf-2.0 -I#{CREW_PREFIX}/include/c++/v1/support/xlocale' \
-           ./configure \
-           --prefix=#{CREW_PREFIX} \
-           --libdir=#{CREW_LIB_PREFIX} \
-           --disable-dependency-tracking \
-           --with-windows-font-dir=#{CREW_PREFIX}/share/fonts/truetype/msttcorefonts \
-           --with-jemalloc \
-           --with-modules \
-           --enable-hdri \
-           --with-perl \
-           --with-rsvg \
-           --with-x"
-    system 'make'
+    system "./configure",
+           "CFLAGS=-I#{CREW_PREFIX}/include/gdk-pixbuf-2.0 -I#{CREW_PREFIX}/include/c++/v1/support/xlocale",
+           "--prefix=#{CREW_PREFIX}",
+           "--libdir=#{CREW_LIB_PREFIX}",
+           "--mandir=#{CREW_MAN_PREFIX}",
```

Comment:
Does `CREW_OPTIONS` work here?

Reply:
No. It works if you remove `--target` from `#{CREW_OPTIONS}`. Don't know why....